### DISCUSSION TITLE
Интеграция в оболочку Windows

### DISCUSSION THREAD

**Discussion by Blackgrowl** [PavelBlend/blender-xray#443] [Ideas]
Добрый.

Я подумал, если это возможно, как я могу интегрировать все эти .obj и .ogf файлы в Blender с помощью PavelBlend?

Я хотел бы сделать так, чтобы каждый раз, когда я открываю один из этих файлов в проводнике Windows, он открывался в Blender и автоматически выполнял импорт.

**Comment by PavelBlend** [comment]
@Blackgrowl я пытался сделать эту возможность, но не получилось. Blender пишет ошибку:
`Error: argument has no '.blend' file extension, not using as new file, exiting!`
Blender даже не запускается. Он требует, чтобы расширение файла было `.blend`.

Это можно проверить, если создать два файла с расширением `bat` и вписать в них примерно такие команды:

**test_blend.bat** файл:
```
"C:\progs\blender2936\blender.exe" "C:\Users\Pavel_Blend\Desktop\test_file.blend"
pause
```

**test_ogf.bat** файл:
```
"C:\progs\blender2936\blender.exe" "C:\Users\Pavel_Blend\Desktop\test_file.ogf"
pause
```

Вторая команда не запустит блендер. В окне будет текст ошибки и блендер закроется.

Невозможно передать блендеру файлы ogf, object из проводника windows. Другого способа это сделать я пока не знаю.
